### PR TITLE
DOCSP-42077 Query History Autocomplete

### PR DESCRIPTION
**Pull request by ajhuh-mdb**
## DESCRIPTION
Adds section in 'View Recent Queries' page on query autocompletion.

## STAGING
https://preview-mongodbajhuhmdb.gatsbyjs.io/compass/DOCSP-24077-query-history-autocompletion/query/recent/

## JIRA
https://jira.mongodb.org/browse/DOCSP-42077

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66ccc46f25dc1b90c70b955c

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by jmd-mongo — APPROVED** [review]
Thanks for these helpful additions, @ajhuh-mdb! This one LGTM.

Thank you,
Joe

File: source/query/recent.txt [hunk 1]
```diff
@@ -33,6 +41,26 @@ You can also hover over a query and perform the following actions:
...
+
+Query Autocomplete Menu 
+-----------------------
+
+If the query bar is empty, you can click the filter bar to open the 
+query autocomplete menu and view up to 50 of your most recent queries. To scroll 
+through your most recent queries, you can use the :kbd:`↑` and :kbd:`↓` arrow 
+keys. If a query is highlighted on the menu, it displays a pop-up panel that 
+shows the filter, project, and sort stages of the query. To select a query from 
+the autocomplete menu, press :kbd:`Enter` or click the query.
```
Comment (betsybutton) [inline source/query/recent.txt:53]: A user can click `Enter` or `Tab` to select the highlighted autocomplete option

**Review by betsybutton — CHANGES_REQUESTED** [review]
Nice work! Just 1 thing that would be good to add. Thanks Ali!

File: source/query/recent.txt (hunk 1 above)
Comment (ajhuh-mdb) [reply source/query/recent.txt]: Updated!

**Comment by ajhuh-mdb** [comment]
Thanks for taking a look @betsybutton ! This should be ready for re-review :)

**Review by betsybutton — APPROVED** [review]
Thanks Ali, LGTM!

**Pull request merged**